GenderPayGap.WebUI/Views/AdminRemoveUser/SuccessfullyRemovedUser.cshtml

```diff
@@ -0,0 +1,73 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model GenderPayGap.WebUI.Models.Admin.AdminRemoveUserViewModel
+
+@{
+    ViewBag.Title = $"Removed user - {Model.UserFullName} - Administration - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@section BeforeMain {
+    @{
+        var crumbs = new List<CrumbViewModel>();
+        crumbs.Add(new CrumbViewModel
+        {
+            Text = "Admin",
+            Href = Url.Action("Home", "Admin")
+        });
+        if (Model.FromViewUserPage)
+        {
+            crumbs.Add(new CrumbViewModel
+            {
+                Text = Model.UserFullName,
+                Href = Url.Action("ViewUser", "AdminViewUser", new {id = Model.UserId})
+            });
+        }
+        else
+        {
+            crumbs.Add(new CrumbViewModel
+            {
+                Text = Model.OrganisationName,
+                Href = Url.Action("ViewOrganisation", "AdminViewOrganisation", new {id = Model.OrganisationId})
+            });
+        }
+        crumbs.Add(new CrumbViewModel
+        {
+            Text = "Remove user from organisation"
+        });
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        <div class="govuk-panel govuk-panel--confirmation govuk-!-margin-bottom-7">
+            <h1 class="govuk-panel__title govuk-!-font-size-36">
+                User has been removed from the organisation
+            </h1>

```

Comment:
When I saw this, I felt like I'd like to be more sure who was removed from which organisation.

Perhaps like this...
![image](https://user-images.githubusercontent.com/7769588/77693091-b2e42200-6f9f-11ea-9cfe-473c52b7103d.png)

```cshtml
<h1 class="govuk-panel__title govuk-!-font-size-36">
    @(Model.UserFullName)
    <br/>
    has been removed from
    <br/>
    @(Model.OrganisationName)
</h1>
```